src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.tsx

```diff
@@ -102,13 +128,13 @@ export const ContactFlowColumn: React.FC<Props> = ({
   };
 
   const handleSelectAll = () => {
+    selectMultipleIds(allContactIds);
     setAnchorEl(null);
-    // TODO implement select all
   };
 
   const handleDeselectAll = () => {
+    deselectMultipleIds(allContactIds);
```

Comment:
Can we not use `deselectAll` from the `useMassSelection` hook here?

Reply:
No as it deselects all contact that are selected not just the ones from this column

Reply:
That makes sense. Thanks!